scripts/dynomite

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
If this script is similar to the one we have in the dynomite repo, then this script should go as a PR to the dynomite repo, not the dynomite-manager repo.